en-GB.json

```diff
@@ -295,6 +295,23 @@
 				"plural": "You have **%d** vote credits. Click the \"Redeem Credits\" button below to redeem them for %d days of premium in this server.",
 				"singular": "You have **%d** vote credit. Click the \"Redeem Credits\" button below to redeem them for %d day of premium in this server."
 			}
+		},
+		"viewstaff": {
+			"title": "Staff",
+			"user_format": "- <@%d> (`%d`)\n",
+			"role_format": "- <@&%d> (`%d`)\n",
```

Comment:
These don't need to be here - They should be actually added to the code rather than using a locale format

```suggestion
```